### ISSUE TITLE
object has no attribute '__qualname__'

### ISSUE THREAD

**Issue by cancan101** [scoutapp/scout_apm_python#596]
I am getting the following error when trying to enable the scout APM on a starlette application:
```
Traceback (most recent call last):
  File "/app/.heroku/python/lib/python3.8/site-packages/uvicorn/protocols/http/httptools_impl.py", line 391, in run_asgi
    result = await app(self.scope, self.receive, self.send)
  File "/app/.heroku/python/lib/python3.8/site-packages/uvicorn/middleware/proxy_headers.py", line 45, in __call__
    return await self.app(scope, receive, send)
  File "/app/.heroku/python/lib/python3.8/site-packages/starlette/applications.py", line 111, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/app/.heroku/python/lib/python3.8/site-packages/starlette/middleware/errors.py", line 181, in __call__
    raise exc from None
  File "/app/.heroku/python/lib/python3.8/site-packages/starlette/middleware/errors.py", line 159, in __call__
    await self.app(scope, receive, _send)
  File "/app/.heroku/python/lib/python3.8/site-packages/scout_apm/async_/starlette.py", line 67, in __call__
    grab_extra_data()
  File "/app/.heroku/python/lib/python3.8/site-packages/scout_apm/async_/starlette.py", line 35, in grab_extra_data
    endpoint.__module__, endpoint.__qualname__
AttributeError: 'GraphQLWithBackground' object has no attribute '__qualname__'
```

`GraphQLWithBackground` is a subclass of `ariadne.asgi.GraphQL`.

**Comment by adamchainz** [comment]
Hi @cancan101 , thanks for the report with traceback. #597 should be a fix. Do you have the ability to try with that patch?

You should be able to use: `pip install https://github.com/scoutapp/scout_apm_python/archive/f3e07cad0318f9b42422a52d0dc6af196fb8bfa6.tar.gz` or place that URL in your requirements file.

**Comment by cancan101** [comment]
Thanks! Yes that fix worked.

**Comment by adamchainz** [comment]
This fix has been released in version 2.18.0: https://pypi.org/project/scout-apm/ . Thanks again for the report.
